react/components/Dots.js

```diff
@@ -0,0 +1,147 @@
+import styles from './styles'
+import debounce from 'debounce'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import React, { Component } from 'react'
+import { resolveSlidesNumber } from '../utils'
+import EventListener from 'react-event-listener'
+
+class Dots extends Component {
+  static propTypes = {
+    /** Classes to style the elements fo the component */
+    classes: PropTypes.shape({
```

Comment:
I think the only way to style things are using "CSS Modules". Allowing to pass arbitrary CSS classes, breaks this model.

Reply:
The idea that I had was that the components that use the Slider was the ones that decide which classes the slider would have, so if you're using the Slider in the shelf for example you just pass some tachyons tokens intead of writing CSS with CSS Modules. I thought of this to be something like we have in [ProductName](https://github.com/vtex-apps/store-components/blob/master/react/components/ProductName/index.js) of `store-components`